test/ModuleInterface/emit-forward-modules.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/Foo.swiftmodule)
+// RUN: %empty-directory(%t/ResourceDir/%target-sdk-name/prebuilt-modules/Foo.swiftmodule)
+// RUN: echo "public func foo() {}" > %t/Foo.swift
```

Comment:
This test has broken the macOS CI:
```
FAIL: Swift(macosx-x86_64) :: ModuleInterface/emit-forward-modules.swift (699 of 13520)
******************** TEST 'Swift(macosx-x86_64) :: ModuleInterface/emit-forward-modules.swift' FAILED ********************
--snip commands--
Command Output (stdout):
--
swiftmodule '/Volumes/swift-ci/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/test-macosx-x86_64/ModuleInterface/Output/emit-forward-modules.swift.tmp/Foo.swiftmodule/x86_64-apple-macos.swiftmodule' is not a forwarding module!

--
Command Output (stderr):
--
<unknown>:0: warning: module interfaces are only supported with Swift language version 5 or later (currently using -swift-version 4)
<unknown>:0: warning: module interfaces are only supported with -enable-library-evolution
```
[Two](https://ci.swift.org/job/swift-PR-osx/22454/console) completely [unrelated](https://ci.swift.org/job/swift-PR-osx/22450/console) pulls fail the CI because of this one test.

Reply:
I attempted to contact some people that can revert this if necessary

Reply:
Thanks.